src/qt/radiobut.cpp

```diff
@@ -39,11 +42,50 @@ bool wxRadioButton::Create( wxWindow *parent,
              const wxString& name)
 {
     m_qtRadioButton = new QRadioButton( parent->GetHandle() );
+
+    if ( (style & wxRB_GROUP) || (style & wxRB_SINGLE) )
+    {
+        JoinNewGroup();
+
+        if( style & wxRB_SINGLE )
+        {
+            // Ensure that other buttons cannot join this group
```

Comment:
```suggestion
            // Ensure that other buttons cannot join the last existing group
```
This is not about _this_ group, but about preventing future buttons from being added to any group that might have been created before this "single button" (which is actually in a group of its own).